corehq/apps/analytics/static/analytix/js/drift.js

```diff
@@ -51,8 +51,39 @@ hqDefine('analytix/js/drift', [
                 hubspot.trackEvent('Identified via Drift');
             });
 
-            $('a[href="#demo-request"]').click(function () {
-                _drift.api.startInteraction({ interactionId: 43079 });
+            $('.shedule-demo-drift-cta').click(function (e) {
```

Comment:
`shedule` => `schedule` (props for being consistent with `shedule` but still, just for everybody's sanity)

Reply:
LOL